doc/rst/platforms/raspberrypi.rst

```diff
@@ -0,0 +1,139 @@
+
+In order to cluster all of the nodes, we must set up password-less SSH. This guide borrows from `MagPi <https://magpi.raspberrypi.com/articles/build-a-raspberry-pi-cluster-computer>`_.
+
+1. Connect all nodes via Ethernet
+
+Connect each node to a network switch via Ethernet cables. For a cluster with only two nodes, you can connect the two nodes directly without the need for a network switch.
+
+2. For each node, set a static IP address in the same subnet.
+
+In this example, we will use ``10.0.0.0`` as our subnet. If your regular network uses this, choose something different like ``192.168.10.0``.
+
+.. code-block:: bash
+
+    sudo nano /etc/dhcpcd.conf
+
+At the end of the file, add the lines:
+
+.. code-block:: bash
+
+    interface eth0
+    static ip_address=10.0.0.1/24
+
+For each node, replace the last digit of the subnet with a new unique value, such as 2, 3, and so on. You should be able to ping each node -- for example, from 10.0.0.1:
+
+.. code-block:: bash
+
+    ping 10.0.0.2
+
+1. Generate SSH keys
+
+On each node, generate an SSH key by running the command:
+
+.. code-block:: bash
+
+    ssh-keygen -t rsa
+
+Press RETURN for each prompt and do not create a passphrase when asked.
+
+4. Copy SSH keys to all nodes in the cluster
+
+For each non-master node, run:
+
+.. code-block:: bash
+
+    ssh-copy-id 10.0.0.1 
+
+to copy that node's SSH key to the master node. Once that has been done on all non-master nodes, copy the master node's SSH key to all other nodes in the cluster by running the same command from the master node, using the IP addresses of the other nodes as the command argument.
+
+5. Set up a shared filesystem
+
+In order for each node to reference the same Chapel executable, the executable must be stored in a shared filesystem that all of the nodes in the cluster have simultaneous access to. There are several ways to do this. 
+
+One way is to mount a flash drive and export it as a network file system to the other nodes in the cluster. The steps to achieve this are described in sections 4.1.1 to 4.1.2 from this `Medium article <https://glmdev.medium.com/building-a-raspberry-pi-cluster-784f0df9afbd>`_.
+
+--------------------------------------
+Configuring for Multi-Locale Execution
+--------------------------------------
+
+For more in-depth information about GASNet or multilocale execution with Chapel,
+refer to the `GASNet documentation`_ and :ref:`readme-multilocale` page,
+respectively.
+
+Ensure that the Chapel compiler is built on each node in your cluster, each node is running the same version of Chapel, and that all nodes in the cluster have SSH access to each other. 
```

Comment:
Perhaps not the compiler -- it is not needed on each node. How about "Ensure that the Chapel tree ...".

What if I build the Chapel tree on the shared file system? Perhaps mention it as a possibility?